### PR TITLE
Replaced partials with components

### PR DESCRIPTION
**Pull request by yohanmishkin**
- Replacements took place in addons/show template
- New components include categories-form, info-form,
    score-detail, and stats-bar.

### CODE REVIEW

File: app/templates/components/categories-form.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<form {{action "save" on="submit"}} class="test-categories-form">
+<form {{action save on="submit"}} class="test-categories-form">
```
Comment (kategengler) [inline app/templates/components/categories-form.hbs]: I think the `isSaving` property later in this file may now be missing

File: app/templates/components/info-form.hbs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<form {{action "save" on="submit"}} class="test-addon-info-form">
+<form {{action save on="submit"}} class="test-addon-info-form">
```
Comment (kategengler) [inline app/templates/components/info-form.hbs]: I think the `isSaving` property in this file may be missing after the move to components.

File: app/components/score-detail.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import Component from '@ember/component';
+
+export default Component.extend({
```
Comment (kategengler) [inline app/components/score-detail.js]: Looks like the template for this component still needs `latestReview` and `isLatestReleaseInLast3Months` and `hasGithubData`

File: app/templates/components/stats-bar.hbs [hunk 4]
```diff
@@ -39,14 +39,14 @@
   </li>
 </ul>
 
-{{#if model.latestTestResult}}
+{{#if testResult}}
   {{#unless isTestResultForLatestVersion}}
```
Comment (kategengler) [inline app/templates/components/stats-bar.hbs]: `isTestResultForLatestVersion` is no longer available

**Comment by yohanmishkin** [comment]
No doubt. Thanks!

**Pull request merged**